drivers/SmartThings/zwave-thermostat/src/qubino-flush-thermostat/init.lua

```diff
@@ -86,9 +86,11 @@ local function configuration_report(driver, device, cmd)
     if configuration_value == 1 then
       device:set_field(CONFIGURED_MODE, COOL_MODE, {persist = true})
       table.insert(supported_modes, capabilities.thermostatMode.thermostatMode.cool.NAME)
+      device:try_update_metadata({profile = "qubino-flush-thermostat-cooling"})
     elseif configuration_value == 0 then
       device:set_field(CONFIGURED_MODE, HEAT_MODE, {persist = true})
       table.insert(supported_modes, capabilities.thermostatMode.thermostatMode.heat.NAME)
+      device:try_update_metadata({profile = "qubino-flush-thermostat"})
```

Comment:
@greens shouldn't "qubino-flush-thermostat" be renamed to "qubino-flush-thermostat-heating" ?

Reply:
I was just going to leave it that way so we would know which was the default.